src/core/ext/client_config/lb_policy_factory.h

```diff
@@ -47,8 +47,16 @@ struct grpc_lb_policy_factory {
   const grpc_lb_policy_factory_vtable *vtable;
 };
 
+typedef struct grpc_lb_policy_address_token {
+  uint8_t *token;
+  size_t token_size;
+} grpc_lb_policy_address_token;
+
 typedef struct grpc_lb_policy_args {
   grpc_resolved_addresses *addresses;
```

Comment:
I was thinking that it would make more sense to add a `bool is_balancer` field to the `grpc_resolved_address` struct, and then add the token separately here.  The reason I say that is that the is_balancer field will be set by the resolver when it returns the addresses, whereas the token is something we get from the balancer itself.  And there are cases where other LB policies will want to see the is_balancer field (if for no other reason than to know that they should ignore that address).

Since the is_balancer change is somewhat orthogonal to the rest of this PR, it can probably be handled in a separate PR.

Does this make sense?